couchpotato/core/media/movie/providers/torrent/iptorrents.py

```diff
@@ -11,12 +11,12 @@ class IPTorrents(MovieProvider, Base):
 
     cat_ids = [
         ([87], ['3d']),
-        ([48], ['720p', '1080p']),
         ([89], ['bd50']),
-        ([96], ['cam', 'ts', 'tc', 'r5', 'scr']),
-		([48, 20, 90], ['brrip']),
+        ([48], ['720p', '1080p']),
+        ([48, 20], ['brrip']),
         ([7, 77], ['dvdrip']),
-        ([6], ['dvdr'])
+        ([6], ['dvdr']),
+        ([96], ['cam', 'ts', 'tc', 'r5', 'scr']),
```

Comment:
No comma needed after the last item in the list.
I believe this is why a user has reported this fix preventing IPTorrent provider from lading/showing...